app/views/repositories/revisions.html.erb

```diff
@@ -26,14 +26,19 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
-<% content_for :action_menu_specific do %>
-  <%= form_tag({:action => 'revision', :id => @project}) do %>
-    <%= l(:label_revision) %>: <%= text_field_tag 'rev', @rev, :size => 8 %>
-    <%= submit_tag 'OK', class: 'button -highlight' %>
+<% content_for :toolbar do %>
+  <%= form_tag({ action: 'revision', id: @project }, { method: :get }) do %>
+    <li class="toolbar-item">
+      <%= label_tag :rev, l(:label_revision), class: 'hidden-for-sighted' %>
+      <%= text_field_tag :rev, @rev, size: 8, placeholder: l(:label_revision) %>
+    </li>
+    <li class="toolbar-item">
+      <%= submit_tag 'OK', class: 'button -highlight' %>
```

Comment:
I see that you are only taking what was there before, but I still ask:

1. shouldn't this be localized?
2. Isn't there a more descriptive label? (e.g. `Show`)